client/my-sites/email/email-management/index.jsx

```diff
@@ -172,7 +172,7 @@ class EmailManagement extends React.Component {
 		}
 
 		const emailForwardingAction = {
-			secondaryAction: translate( 'Add Email Forwarding' ),
+			secondaryAction: translate( 'Add email forwarding' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.